content/docs/start/index.md

```diff
@@ -48,13 +48,15 @@ Changes to be committed:
 $ git commit -m "Initialize DVC"
 ```
 
-DVC features can be grouped into functional components. We'll explore them one
-by one in the next few sections:
+Now you're ready to DVC!
```

Comment:
This p helps close the previous section (install/init steps). Note that the previous p is preserved too (lines 53-4).